src/AccessibilityInsights/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="CommandLineParser" version="2.5.0" targetFramework="net471" />
```

Comment:
Do we need to update thirdpartynotices.html or do any OSS registration related work?

Reply:
updated thirdpartynotices.html